### ISSUE TITLE
Render causes ZASF to lose controller focus.

### ISSUE THREAD

**Issue by mikobuntu** [LMMS/lmms#873]
Im not even sure if this bug is entirely related to just the latest stable1.1 branch, there could be others too...

Anyway to reproduce connect for example the frequency knob on the ZynAddSubFX gui ( LMMS plugin gui obviously ) to the LFO Controller and render a short song. 
Now you will notice that as the song is started to render the effect of the controller is not passed to the audio, which shows this happens at the immediate point of render beginning.And also note the actual switch in the /connection settings/user controller is now switched off.

_note others such as instrument pan and effect for other instruments etc are not effected by this bug/regression_ i can do more testing if required *

from the cli i get the error below:-

ALSA-sequencer: unhandled input event 66
ALSA-sequencer: unhandled input event 66
Starting Audio: NULL
Audio Started
Starting MIDI: NULL
MIDI Started

**Comment by Umcaruje** [comment]
@mikobuntu can you reproduce this on the current master branch, and on @curlymorphic's zyn branch (don't know the exact name)?

**Comment by tresf** [comment]
IIRC, zynwin should be a safe branch to compile.  https://github.com/curlymorphic/lmms/tree/zynwin

**Comment by curlymorphic** [comment]
> IIRC, zynwin should be a safe branch to compile. https://github.com/curlymorphic/lmms/tree/zynwin

Yes that is the current branch.

**Comment by tresf** [comment]
More specifically... 

```bash
git clone -b zynwin https://github.com/curlymorphic/lmms
```

**Comment by mikobuntu** [comment]
ok I'm cloning @curlymorphic 's branch now , I will build and test the aforementioned bug and also test with master next and report back guys ;)

**Comment by mikobuntu** [comment]
@curlymorphic  my Liblo-dev package is not up to date ( running ubuntu-12.04 on this machine ) will not having OSC enabled affect my results? thanks

**Comment by curlymorphic** [comment]
@mikobuntu 

sudo apt - get install liblo-dev liblo7 should work fine on 12.04, 

if you are running into problems, can you link me the results of running cmake please.

**Comment by mikobuntu** [comment]
@curlymorphic @tresf @Umcaruje   Ok I was able to build the zynwin branch and on testing the bug still exists. I will test this further and report back any findings .

**Comment by SecondFlight** [comment]
Leaving a comment here to remind myself to try reproducing when I get home.

**Comment by SecondFlight** [comment]
Closing, as I was not able to reproduce this on 1.2 RC8.

Steps I took to reproduce:
1. Add Zyn to an empty project.
2. Change Zyn to play a square wave instead of a sine wave so the filter would be more audible.
3. Add a LFO controller.
4. Increase the LFO speed (because why not)
5. Connect the FREQ knob on the LMMS Zyn interface to the new LFO controller.
6. Add a single 4-bar note to the Zyn track.
7. Render the song to WAV using the default settings (this is a fresh LMMS install).

I was able to hear the filter in the rendered audio.
